i18n/execute.py

```diff
@@ -31,15 +31,15 @@ def call(command, working_directory=config.BASE_DIR):
     LOG.info(command)
     proc = sp.Popen(command, stdout=sp.PIPE, stderr=sp.PIPE, cwd=working_directory, shell=True)
     out, err = proc.communicate()
-    return (out, err)
+    return (out.decode(), err.decode())
```

Comment:
`out` and `err` are byte strings. I looked at the six module but I didn't see an obvious way to deal with this issue. 

I decided that it would probably be safest to have `call` continue to return byte strings, since it can be used to run arbitrary commands which may produce output in different encodings. We can decode the strings as is necessary when `call` is invoked, which currently only seems to be the `i18n/validate.py` script.